src/main/java/org/aion/wallet/connector/api/ApiBlockchainConnector.java

```diff
@@ -92,14 +92,19 @@ public BigInteger getBalance(String address) {
         return API.getChain().getBalance(new Address(address)).getObject();
     }
 
+    @Override
+    public int getPeerCount() {
+        return 0;
```

Comment:
can be replaced with 
`return ((List) API.getNet().getActiveNodes().getObject()).size();`